bucoffea/vbfhinv/vbfhinvProcessor.py

```diff
@@ -168,9 +168,10 @@ def process(self, df):
         leadak4_pt_eta = (diak4.i0.pt > cfg.SELECTION.SIGNAL.LEADAK4.PT) & (np.abs(diak4.i0.eta) < cfg.SELECTION.SIGNAL.LEADAK4.ETA)
         trailak4_pt_eta = (diak4.i1.pt > cfg.SELECTION.SIGNAL.TRAILAK4.PT) & (np.abs(diak4.i1.eta) < cfg.SELECTION.SIGNAL.TRAILAK4.ETA)
         hemisphere = (diak4.i0.eta * diak4.i1.eta < 0).any()
+        has_track = np.abs(diak4.i0.eta) <= 2.5
 
-        leadak4_id = diak4.i0.tightId & (diak4.i0.chf > cfg.SELECTION.SIGNAL.LEADAK4.CHF) &  (diak4.i0.nhf < cfg.SELECTION.SIGNAL.LEADAK4.NHF)
-        trailak4_id = diak4.i1.tightId & (diak4.i1.chf > cfg.SELECTION.SIGNAL.TRAILAK4.CHF) &  (diak4.i1.nhf < cfg.SELECTION.SIGNAL.TRAILAK4.NHF)
+        leadak4_id = diak4.i0.tightId & (has_track*((diak4.i0.chf > cfg.SELECTION.SIGNAL.LEADAK4.CHF) & (diak4.i0.nhf < cfg.SELECTION.SIGNAL.LEADAK4.NHF)) + ~has_track)
+        trailak4_id = (diak4.i1.chf > cfg.SELECTION.SIGNAL.TRAILAK4.CHF) & (diak4.i1.nhf < cfg.SELECTION.SIGNAL.TRAILAK4.NHF)
```

Comment:
Shouldnt this CHF cut also be protected by `has_track`?

Reply:
I think for the trailing jet, the cfg values are dummy variables (e.g. CHF is -999 and NHF is 999, I believe). So every trailing jet is passing the requirement. 

So I didn't include the has_track for that one, but for completeness I can also include it if you like me to do so.